apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/TransactionPool.cs

```diff
@@ -39,6 +39,8 @@ public static class TransactionPool
         // Holds a list of all active transactions with back pointers to the owning session.
         static readonly ConcurrentDictionary<ByteString, Session> s_activeTransactionTable = new ConcurrentDictionary<ByteString, Session>();
 
+        private static Logger Logger { get; } = Logger.DefaultLogger;
```

Comment:
Again, this won't see any changes to `Logger.DefaultLogger` - is that intentional? (More of a concern than the SpannerConnection one, as it's part of type initialization.)

Reply:
There were some code paths where it did not make sense to log messages via an instance -- shared pools were one case.  So right now these will not be seen from EF -- I might revisit this, but it could be odd to see some messages from a pool show up in one EF logger and other messages in another logger.